README.rst

```diff
@@ -50,8 +50,7 @@ Manifest
 
 9.  Flexible storage capabilities. Built-in
     ``simp_le -f fullchain.pem    -f key.pem``,
-    ``simp_le -f chain.pem -f cert.pem -f key.pem``, etc. Extensions
-    through ``simp_le -f external.sh``.
+    ``simp_le -f chain.pem -f cert.pem -f key.pem``, etc.
```

Comment:
Let's just remove this list item entirely; without `ExternalIOPlugin` I don't think it's fair to say that we have "flexible storage capabilities."

Reply:
There is still a bit of flexibility regarding how the private key / cert / chain are stored to disk. Is it enough to keep the line as-is or do you still prefer it to be removed ?